Classes/Aggregate/NewContentElementWizardAggregate.php

```diff
@@ -52,6 +52,16 @@ class NewContentElementWizardAggregate extends AbstractAggregate implements Lang
     protected $pageTSConfigFilePath = 'Configuration/PageTSconfig/';
 
     /**
+     * @var array
+     */
+    protected $icons = [];
+
+    /**
+     * @var array
+     */
+    protected $iconNames = [];
+
```

Comment:
There is no need to add two extra properties here. Just take one associative array.